### PR TITLE
MBS-10400 Convert Edit Release Group edit to React

### PR DESCRIPTION
**Pull request by anirudhjain75**

### CODE REVIEW

File: root/edit/details/EditReleaseGroup.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import DescriptiveLink from
+  '../../static/scripts/common/components/DescriptiveLink';
+import Diff from '../../static/scripts/edit/components/edit/Diff';
+import ExpandedArtistCredit from
+  '../../static/scripts/common/components/ExpandedArtistCredit';
+import WordDiff from '../../static/scripts/edit/components/edit/WordDiff';
+import FullChangeDiff from
+  '../../static/scripts/edit/components/edit/FullChangeDiff';
+
+type EditReleaseGroupProp = {
```
Comment (reosarevok) [inline root/edit/details/EditReleaseGroup.js]: Props

File: root/edit/details/EditReleaseGroup.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  },
+};
+
+const EditReleaseGroup = ({edit}: {edit: EditReleaseGroupProp}) => {
+  const display = edit.display_data;
+  const name = display.name;
+  const comment = display.comment;
+  const type = display.type;
+  const secondaryTypes = display.secondary_types;
+  console.log(display);
```
Comment (reosarevok) [inline root/edit/details/EditReleaseGroup.js]: Remove

File: root/edit/details/EditReleaseGroup.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        <Diff
+          label={l('Secondary Types:')}
+          newText={secondaryTypes.new}
+          oldText={secondaryTypes.old}
+          split=" \+"
+        />
+      ) : null}
+      {display.artist_credit ? (
+        <tr>
+          <th>{l('Artist:')}</th>
```
Comment (reosarevok) [inline root/edit/details/EditReleaseGroup.js]: addColonText should work fine here :)

File: root/edit/details/EditReleaseGroup.js [hunk 4]
```diff
@@ -0,0 +1,94 @@
+/*
+ * @flow
+ * Copyright (C) 2020 Anirudh Jain
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (reosarevok) [reply root/edit/details/EditReleaseGroup.js:10]: This is still using the old import format :)

**Comment by yvanzo** [comment]
@brainzbot, retest this please.

**Comment by yvanzo** [comment]
@brainzbot, retest this please.

**Pull request merged**